src/components/NFTproduct/Nftproduct.tsx

```diff
@@ -1,121 +1,120 @@
-  const [isReadMore, setIsReadMore] = useState(false)
+  const { t } = useTranslation()
+  const [isReadMore, setIsReadMore] = useState('')
 
-  const showScroll = (res: boolean) => {
-    setIsReadMore(res)
+  const showScroll = (readMore: boolean) => {
+    readMore? setIsReadMore('scroll') : setIsReadMore('')
   }
 
   return (
-    <NFTS container direction="column">
+    <Container>
       <NFTsHeader />
-      <Grid className={classes.container} container direction="row" justify="center">
-        {/* left */}
-        {/* <Grid item> */}
-        <div>
-          <Grid container direction="column">
-            <Card className={classes.img}>
-              <div>
-                <img src={Vector} />
-                <p>1234 Views</p>{' '}
-              </div>
-
-              <CardMedia className={classes.img} component="img" image={image} title="IMG" />
-            </Card>
-
-            {/* created by */}
-            <Card className={classes.created}>
-              <div className={classes.created_title}>
-                <p>
-                  Details created By<span>Muhammed Amin</span>
-                </p>
-              </div>
-
-              <div className={classes.created_text}>
-                <p>
-                  Put your NFT assets up as collateral for a loan, or offer loans to other users on their non-fungible
-                  tokens Put your NFT assets up as collateral for a loan, or offer loans to other users on their
-                  non-fungible tokens
-                </p>
-              </div>
-            </Card>
-          </Grid>
-        </div>
-
-        {/* </Grid> */}
-
-        {/* right */}
-        <div>
-          <Grid container direction="column">
-            <div className={classes.title}>
-              <p>Apple Watch Series 4 GPS</p>
+      <Grid>
+        <LeftGrid>
+          <ImgCard>
+            <img src={Rectangle} />
+            <p>1234  {t('views')}</p>
+          </ImgCard>
+          <LeftTextCard>
+            <CreatedTitle>
+              <p>
+              {t('createdBy')}<span>Muhammed Amin</span>
+              </p>
+            </CreatedTitle>
+            <CreatedText>
+              <p>
+                Put your NFT assets up as collateral for a loan, or offer loans to other users on their non-fungible
+                tokens Put your NFT assets up as collateral for a loan, or offer loans to other users on their
+                non-fungible tokens
+              </p>
+            </CreatedText>
+          </LeftTextCard>
+        </LeftGrid>
+        <RightGrid>
+          <RightTitle>
+            <p>Apple Watch Series 4 GPS</p>
+          </RightTitle>
+          <RightSubTitle>{t('prediction')}: Round 11 (Bronze) - Only 100 Available</RightSubTitle>
+          <PublisherCard height='91px'>
+            <div>
+              <p>
+              {t('publisher')} :<span>Muhammed Amin</span>
+              </p>
+              <p>8% Percentage on each reselling transaction</p>
```

Comment:
only the number will be given add localization

Reply:
not  completed,